user/tests.py

```diff
@@ -205,3 +205,36 @@ def test_home_without_login(self):
     self.assertEqual(home_response.status_code, 302)
 
 
+class TweetSucceseTest(TestCase):
+  def setUp(self):
+    Account.objects.create_user(username='people', email='test@test.test', password='testpassword', age='1')
+    self.client.login(username='people', password='testpassword')
+    self.tweet_url = reverse('tweet')
+    self.tweet_data = {'text':'test'}
+    self.home_url = reverse('home')
+
+  def test_tweet_succese(self):
+    self.tweet_response = self.client.post(self.tweet_url, self.tweet_data)
+    self.assertRedirects(self.tweet_response, self.home_url)
+    tweet_number = Tweet.objects.count()
+    self.assertEqual(tweet_number, 1)
+  
+
+class TweetFailTest(TestCase):
+  def setUp(self):
+    Account.objects.create_user(username='people', email='test@test.test', password='testpassword', age='1')
+    self.tweet_url = reverse('tweet')
+
+  def test_tweet_fail(self):
+    self.client.login(username='people', password='testpassword')
+    tweet_data = {'text':''}
+    tweet_response = self.client.post(self.tweet_url, tweet_data)
+    self.assertRedirects(tweet_response, self.tweet_url)
+    tweet_number = Tweet.objects.count()
+    self.assertFalse(tweet_number)
```

Comment:
```suggestion
    self.assertEqual(Tweet.objects.count(), 0)
```
[nits]
このほうがツイートが存在しない(0個である)ことが明確でわかりやすいな〜と思いました！